README.md

```diff
@@ -56,17 +56,17 @@ The following is a list of links to potential starting points:
 
 * [Diagrams](diagrams/) - various generated diagrams.
   The PlantUML file [`diagrams/metametamodel-gen.puml`](diagrams/metametamodel-gen.puml) is generated from the meta-circular [self-definition of `lioncore`](src-pkg/m3/lioncore.ts).
-  This generated PlantUML file can then be compared with [this one](https://github.com/LIonWeb-org/organization/blob/main/metametamodel/metametamodel.puml): they should have exactly the same contents apart from a couple of obvious differences.
+  This generated PlantUML file can then be compared with [this one](https://github.com/LionWeb-org/organization/blob/main/metametamodel/metametamodel.puml): they should have exactly the same contents apart from a couple of obvious differences.
```

Comment:
Can we rename the organization?

Reply:
Looks like it: https://docs.github.com/en/organizations/managing-organization-settings/renaming-an-organization